__tests__/minimize.ts

```diff
@@ -18,6 +18,8 @@ test('Parse & stringify statements', () => {
 		'query.position()': 'q.position()',
 		'variable.x = 1; return variable.x;': 'v.v0=1;return v.v0;',
 		'20 + 50': '70',
+		'variable.x + 0' : 'v.v0',
+		'0 + variable.x' : 'v.v0'
```

Comment:
Unrelated to this PR but we might want to look into only changing the var names if they're longer than 1 char